gulp/sassStyles.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-env es6 */
+'use strict';
+
+// External dependencies
+import {src, dest} from 'gulp';
+import pump from 'pump';
+import requireUncached from 'require-uncached';
+
+// Internal dependencies
+import {paths, gulpPlugins} from './constants';
+
+/**
+ * Sass, if that's being used.
+ */
+export default function sassStyles(done) {
+    // get a fresh copy of the config
+   const config = requireUncached(paths.config.themeConfig);
```

Comment:
@hellofromtonya do you think using `requireUncached` is necessary? It might be cleaner to read the config once in constants, export it and then import it here rather than break cache to re-read.

I'm not sure if we expect the debug config to be toggled while gulp is running. Reading uncached each time, like is happening now, will allow that to happen.

Reply:
I'd say yes, @ataylorme.  You can be working along and then discover: "Whoops, I misspelled the prefix or theme name."  Right? That can happen.  In that case, you want the ability to change the theme configuration on-the-fly with gulp running, and have it rebuild for you.

Reply:
Makes sense, I just wanted to confirm.